core/message-tags-3.3.md

```diff
@@ -105,7 +105,8 @@ Servers MAY apply moderation to this command using existing or newly specified m
 
 Servers MUST NOT deliver `TAGMSG` to clients that haven't negotiated the message tags capability.
 
-Servers SHOULD reject any `TAGMSG` command sent without tags. In this case, they MUST use the `ERR_NEEDMOREPARAMS` (`461`) error numeric.
+Servers SHOULD reject any `TAGMSG` command sent without tags, or where all tags on the message end up
+being removed by the server. In this case, they MUST use the `ERR_NEEDMOREPARAMS` (`461`) error numeric.
```

Comment:
I've amended it to say MUST for both. I don't think we should specify this as optional behaviour for servers. Do any servers not respond to a PRIVMSG without text?